mock/mock.go

```diff
@@ -209,6 +209,12 @@ func (m *Mock) On(methodName string, arguments ...interface{}) *Call {
 	return c
 }
 
+// IsCallExpected checks if a method call is expected.
+func (m *Mock) IsCallExpected(method string, arguments ...interface{}) bool {
+	found, _ := m.findExpectedCall(method, arguments...)
+	return found >= 0
+}
```

Comment:
OK, while I'm not convinced, I will comply. Will update this PR to remove IsCallExpected(). @ernesto-jimenez, thank you for your time spend on this, appreciated!